src/main/java/com/funnyboyroks/mapify/util/Util.java

```diff
@@ -322,4 +323,20 @@ public static boolean dimsMatch(Point dims, String max) {
 
         return area <= 0 || dims.x * dims.y <= area;
     }
+
+    /**
+     * Remove all old maps from the plugin data
+     */
+    public static void removeOldMaps() {
+        // Clear the map data
+        Mapify.INSTANCE.dataHandler.data.mapData.clear();
+        // Save the changes
+        Mapify.INSTANCE.dataHandler.dirty();
+        try {
+            Mapify.INSTANCE.dataHandler.saveData(true);
+        } catch (IOException e) {
+            Mapify.INSTANCE.getLogger().severe("Error saving data after removing old maps:");
+            e.printStackTrace();
+        }
+    }
```

Comment:
This does not solve the issue.  We don't want to remove all old maps (that will break Mapify every time the server is restarted).  We want a method which will allow broken maps to be removed.